webextension/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    webextensions: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:mozilla/recommended",
+  ],
+  plugins: [
+    "json",
+    "mozilla",
+  ],
+  root: true,
+  rules: {
+    "eqeqeq": "error",
+    "indent": ["error", 2],
+    "no-console": "warn",
+    "no-var": "error",
```

Comment:
You can delete this rule, since you're overriding it with `"no-var": 0` below (L27).

Reply:
Done